packages/website/src/DocModel/comment/CommentNodeContainer.ts

```diff
@@ -1,20 +1,19 @@
 import type { ApiItem, ApiModel } from '@microsoft/api-extractor-model';
 import type { DocNodeContainer } from '@microsoft/tsdoc';
 import { createCommentNode } from '.';
-import { CommentNode } from './CommentNode';
+import { AnyDocNodeJSON, DocNodeJSON, node } from './CommentNode';
 
-export class CommentNodeContainer<T extends DocNodeContainer = DocNodeContainer> extends CommentNode<DocNodeContainer> {
```

Comment:
ci is falling because you've removed this class from this file but this class is used in `./src/DocModel/DocEnum.ts:3:10`

Reply:
Thanks